components/story-elements/table.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import papa from "papaparse";
+import ReactTable from "react-table";
+import _ from "lodash";
+
+class Table extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tableData: []
+    };
+    this.parseCSVToJson = this.parseCSVToJson.bind(this);
```

Comment:
Is this bind necessary?

Reply:
Yes, because in `parseCSVToJson ` function, `this.setState` is being used. If there is no binding, the value of `this` will be undefined.

Reply:
👍